CAM-12326: configure nullable field for array and dto properties and enable nullable by default (#1097)
camunda/camunda-bpm-platform: engine-rest/engine-rest-openapi/README.md

#### Nullable

> By default the properties of type `boolean`, `string`, 	`array`, and `dto` has `nullable` field enabled by default to ensure ensures work with C# clients.

Reviewer: Some typos here:
```suggestion
By default, the properties of type `boolean`, `string`, `array`, and `dto` have `nullable` field enabled by default to ensure that the OpenAPI specification can work with C# clients.
```
